SPECS/yaml-cpp/yaml-cpp.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "yaml-cpp-0.7.0.tar.gz": "43e6a9fcb146ad871515f0d0873947e5d497a1c9c60c58cb102a97b47208b7c3"
+  "0.8.0.tar.gz": "fbe74bbdcee21d656715688706da3c8becfd946d92cd44705cc6098bb23b3a16"
```

Comment:
Please use the previous format of the file name. Using only the version number has a high chance of causing conflicts in our sources blobstorage.

Reply:
And in order to keep the spec building, you'll also need to apply a change I've added for the `Source0` tag in the spec.

Reply:
Pipeline Run: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=494412&view=results

@PawelWMS - Does this work now and could you please take a look